test/unit/math/calculation.js

```diff
@@ -57,27 +57,19 @@ suite('Calculation', function() {
       });
       test('should return correct distance', function() {
         result = dist(0, 0, 2, 3);
-        assert.equal(result, 3.6055512754639896); // Math.hypot(2, 3)
+        assert.equal(result, 3.605551275463989); 
       });
       test('should return positive  distance', function() {
         result = dist(0, 0, -2, -3);
-        assert.equal(result, 3.6055512754639896); // Math.hypot(2, 3)
+        assert.equal(result, 3.605551275463989); 
       });
       test('should return correct distance', function() {
         result = dist(0, 0, 0, 2, 3, 5);
-        assert.equal(result, 6.164414002968977); // Math.hypot(2, 3, 5)
+        assert.equal(result, 6.164414002968976); 
       });
       test('should return positive  distance', function() {
         result = dist(0, 0, 0, -2, -3, 5);
-        assert.equal(result, 6.164414002968977); // Math.hypot(2, 3, 5)
-      });
-      test('should not underflow', function() {
```

Comment:
I think we want to have these two tests in there, and make sure that the function behaves similarly to Math.hypot() (which does not over or under flow)